api/controllers/StatsController.js

```diff
@@ -1,8 +1,13 @@
 const dayjs = require('dayjs');
 const isBetween = require('dayjs/plugin/isBetween');
+const utc = require('dayjs/plugin/utc');
```

Comment:
You should pull these out in to a utility somehow or some sort of global. Your code is super wet.